thewalrus/quantum.py

```diff
@@ -659,6 +659,70 @@ def gen_Qmat_from_graph(A, n_mean):
     return Q
 
 
+def photon_number_var(mu, cov, j, k, hbar=2):
+    r""" Calculate the variance/covariance of the photon number distribution
+    of a Gaussian state.
+
+    Last two eq. of Part II. in 'Multidimensional Hermite polynomials and
+    photon distribution for polymode mixed light', Dodonov et al.
+    (https://journals.aps.org/pra/abstract/10.1103/PhysRevA.50.813).
+
+    Args:
+        mu (array): mean <Q> in ref. above consisting of [q1, q2, ..., qn, p1, p2, ..., pn]
+        cov (array): the covariant matrix of the gaussian state
+        j (int): the first index of the covariance matrix
+        k (int): the second index of the covariance matrix

```

Comment:
It might be worth re-wording these, so that it doesn't get confused with the covariance matrix of the Gaussian state `cov`.